index.html

```diff
@@ -45,12 +45,34 @@ <h1>This Test of GeoJSON with Leaflet</h1>
 			'Imagery © <a href="http://mapbox.com">Mapbox</a>',
 		id: 'mapbox.light'
 	}).addTo(map);
+	
+	// get color depending on population density value
+	function getColor(d) {
+		return d > 8000 ? '#800026' :
+				d > 7000  ? '#BD0026' :
+				d > 6000  ? '#E31A1C' :
+				d > 5000  ? '#FC4E2A' :
+				d > 4000   ? '#FD8D3C' :
+				d > 3000   ? '#FEB24C' :
+				d > 2000   ? '#FED976' :
+							'#FFEDA0';
+	}
+
+	function style(feature) {
+		return {
+			weight: 2,
+			opacity: 1,
+			color: 'white',
+			dashArray: '3',
+			fillOpacity: 0.7,
+			fillColor: getColor(feature.properties.density)
+		};
+	}
 
-L.geoJSON(someFeatures, {
-    filter: function(feature, layer) {
-        return feature.properties.show_on_map;
-    }
-}).addTo(map);
```

Comment:
Removed to mimic the source code from Leaflet Choropleth map tutorial

http://leafletjs.com/examples/choropleth/